sunkit_image/utils/noise.py

```diff
@@ -55,9 +55,9 @@ def noise_estimation(img, patchsize=7, decim=0, confidence=1 - 1e-6, iterations=
         [1., 1., 1., ..., 1., 1., 0.],
         [0., 0., 0., ..., 0., 0., 0.]])
     >>> estimate['nlevel'] # Prints nlevel
-    array([0.97398633])
+    array([0.97470748])
```

Comment:
Do we know why these changeg?

Reply:
When i ran the tests locally the docs failed since the expected value is different from the gotten value